doc/welcome.rst

```diff
@@ -0,0 +1,21 @@
+Welcome
+=======
+
+Welcome to the Cabal user guide.
+Cabal is the "common architecture for building applications and tools" and consists of a fileformat for describing Haskell packages, a library for working with those files, and a commandline tool for downloading, building and running Haskell applications and libraries.
+
+Finding Help
+------------
+
+If you are new to Haskell and Cabal, then you can find help in the `Haskell Matrix <https://matrix.to/#/#haskell:matrix.org>`__ chat or on the official `Haskell Discourse <https://discourse.haskell.org>`__ forum.
+
+If you want to contact the maintainers and developers of cabal, then you can join the `Cabal and Hackage Matrix <https://matrix.to/#/#hackage:matrix.org>`__ chat.
+If you prefer to use mailing lists, then you can follow `libraries@haskell.org <https://mail.haskell.org/mailman/listinfo/libraries>`__ for general discussion and queries about the library ecosystem, and `cabal-devel@haskell.org <https://mail.haskell.org/mailman/listinfo/cabal-devel>`__ for the development of Cabal.
+
+Reporting Issues and Requesting Features
+----------------------------------------
+
+Cabal is developed on `GitHub <https://github.com/haskell/cabal>`__.
+Please report any flaws or feature requests on the `issue
+tracker <https://github.com/haskell/cabal/issues>`__.
+If you want to start contributing to Cabal yourself, then you can find help in the `CONTRIBUTING.md <https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md>`__ in the repository.
```

Comment:
What do you think about moving this section into a separate guide page like https://github.com/haskell/cabal/pull/9379/files#diff-495e1081527403be1d72a3ae8c14fa5cfa19754bffd27c7b6f4455fa589d1836? It would help with SEO to have the page slug also mention "how-to-get-help-report-issues".